test/spec/GeoExt/data/store/WfsFeatures.test.js

```diff
@@ -0,0 +1,166 @@
+Ext.Loader.syncRequire(['GeoExt.data.store.WfsFeatures']);
+
+describe('GeoExt.data.store.WfsFeatures', function() {
+
+    describe('basics', function() {
+        it('is defined', function() {
+            expect(GeoExt.data.store.WfsFeatures).not.to.be(undefined);
+        });
+    });
+
+    describe('constructor (no arguments)', function() {
+
+        it('raises an error', function() {
+            var errRaised = false;
+            try {
+                Ext.create('GeoExt.data.store.WfsFeatures');
+            } catch (e) {
+                errRaised = true;
+            }
+
+            expect(errRaised).to.be(true);
+        });
+    });
+
+    describe('configs and properties', function() {
+        var store;
+        beforeEach(function() {
+            store = Ext.create('GeoExt.data.store.WfsFeatures', {
+                url: 'foo'
```

Comment:
Very minor: This adds a warning to the output of  `npm test`. We can also use the URL from the `loading without paging` test, can't we?